### PR TITLE
Sprint 1 issue #6 Resolved

### PR DESCRIPTION
**Pull request by Matthew-Clendenning**
This pull request resolves issue #6 

What was changed: Wrote documentation about what TPEN, TPEN API, and TPEN data services are. I also made changes to the styling on landing page.

Why was it changed: Added documentation so users can have a clear understanding of what TPEN is and how to use it. Style changes to the landing page was necessary for mobile users to access the website easily.

How was it changed: I created a README.md file with the information about TPEN. Edited the index.html file and added a hamburger menu for mobile users to access different pages of the site, username and password input boxes, and a submit button. Added media queries to the css file to make the site responsive. Created a small JavaScript file for the hamburger menu as well as logic for the username and password for testing purposes.

### CODE REVIEW

File: components/classroom/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+1. **Create an Account**: Sign up for a T-PEN account on the website https://t-pen.org/TPEN/
+2. **Upload Manuscripts**: Upload digital images of manuscripts you wish to transcribe.
+3. **Start Transcribing**: Use the transcription interface to begin transcribing the text.
+4. **Annotate and Collaborate**: Utilize the annotation tools and invite collaborators to join your project.
+5. **Export Data**: Once your transcription is complete, export the data in your preferred format.
+
+## User Guide Best Practices
+
+* `index.html`: The default interface to launch,
+* TPEN interface suuports many files e.g. `*js`, `*.css`, `*.html`, etc.,
```
Comment (camilledaugherty) [inline components/classroom/README.md:36]: supports is misspelled

File: components/classroom/index.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        <div class="header">
+            <h1>TPEN Classroom Interface</h1>
+            <div class="hamburger-menu" onclick="toggleMenu()">
+                &#9776 <!-- Hamburger icon -->
+            </div>
+        </div>
+            <div class ="navbar" id="navbar">
+            <!--navigation bar below: we can try making this collapsible on mobile devices?
+            placeholder categories, will be changed as we solidify the pages needed
+            -->
```
Comment (camilledaugherty) [inline components/classroom/index.html:19]: you can remove my comment! i didn't realize you were going to do the collapsible menu

File: components/classroom/index.html [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+                <li><a href="#annotations">Annotations</a></li>
+                <li><a href="#groups">Groups</a></li>
+                <li><a href="#contact">Contact</a></li>
+            </ul>
+            </div>
+
+            <div class="page-container">
+                <h1>Welcome!</h1>
+                <div class="input-container">
+                    <h2>Sign Up/In</h2>
```
Comment (camilledaugherty) [inline components/classroom/index.html:31]: do we want to have two separate forms for signing up and logging in? if this works that's fine, just wondering bc signup might need more information from the user e.g. email address

File: components/classroom/styles.css [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+    width: 100%;
+}
+
+/* Hamburger icon */
+.hamburger-menu {
+    display: none;
+    font-size: 25px;
+    cursor: pointer;
+    position: absolute;
+    top: 20px;
```
Comment (camilledaugherty) [inline components/classroom/styles.css:92]: super nitpicky but would look better if it was aligned with the website header! so top 30px instead of 20px

**Comment by kate-holdener** [comment]
> This pull request resolves issue #6
> 
> What was changed: Wrote documentation about what TPEN, TPEN API, and TPEN data services are. I also made changes to the styling on landing page.
> 
> Why was it changed: Added documentation so users can have a clear understanding of what TPEN is and how to use it. Style changes to the landing page was necessary for mobile users to access the website easily.
> 
> How was it changed: I created a README.md file with the information about TPEN. Edited the index.html file and added a hamburger menu for mobile users to access different pages of the site, username and password input boxes, and a submit button. Added media queries to the css file to make the site responsive. Created a small JavaScript file for the hamburger menu as well as logic for the username and password for testing purposes.

Please use:
Fixes #6 
to automatically close the issue when the PR is merged.

**Comment by kate-holdener** [comment]
@Matthew-Clendenning I see that you have the commit from @camilledaugherty in your branch. I'm confused on how it got here.

**Comment by Matthew-Clendenning** [comment]
Hi Dr. Holdener,

I messaged our tech-lead Sowmya prior to sprint 1 because I was confused on how to edit the styling of our landing page that Camille made. She replied saying I needed to wait for her to finish her work and fetch the changes she made within her branch. I used git fetch to do so. I hope this didn't cause any issues.

Thank you,
Matthew Clendenning
________________________________

**Review by MutyaSowmya123 — CHANGES_REQUESTED** [review]
Sign Up/In is not required just remove that:
`<h2>Sign Up/In</h2>`

**Comment by Matthew-Clendenning** [comment]
Reviewed comments made by team and made the required changes.

**Review by MutyaSowmya123 — APPROVED** [review]

**Pull request merged**